R/tlf_ae_specific.R

```diff
@@ -75,16 +77,39 @@ tlf_ae_specific <- function(outdata,
   n_col <- ncol(tbl)
 
   # Define title
-  title <- collect_title(outdata$meta, outdata$population, outdata$observation, outdata$parameter, analysis = "ae_specific")
-  footnotes <- vapply(footnotes, glue::glue_data,
+  if(is.null(title)){
+    title <- collect_title(outdata$meta,
+      outdata$population,
+      outdata$observation,
+      outdata$parameter,
+      analysis = "ae_specific")
+  }
+
+ footnotes <- vapply(footnotes, glue::glue_data,
     .x = list(medra_version = medra_version), FUN.VALUE = character(1)
   )
   names(footnotes) <- NULL
 
   # Define column header
+
+  col_tbl <- strsplit(names(tbl), "_") |>
```

Comment:
Should take into account the reality of the tbl that is passed in, if we're not going to be passing this info explictly in the outdata. Currently this only takes into account and pays attention to "within" variables. would need to update further for the "between" variables `diff`, `diff_p` and `diff_ci`.